test/locals_test.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+# This test is going to use the RubyVM::InstructionSequence class to compile
+# local tables and compare against them to ensure we have the same locals in the
+# same order. This is important to guarantee that we compile indices correctly
+# on CRuby (in terms of compatibility).
+#
+# There have also been changes made in other versions of Ruby, so we only want
+# to test on the most recent versions.
+return if !defined?(RubyVM::InstructionSequence) || RUBY_VERSION < "3.2"
+
+require "yarp_test_helper"
+
+class LocalsTest < Test::Unit::TestCase
+  invalid = []
+  todos = []
+
+  # Invalid break
```

Comment:
I wonder if instead of this, or maybe as an add-on PR, we should make these test snippets smaller such that they're all valid?

Reply:
I think maybe a separate PR. It's going to be a non-trivial amount of work because a lot of them are coming directly from other imported sources.